src/ratbag_emu/protocol/base.py

```diff
@@ -29,10 +28,10 @@ def __init__(self, owner, rdesc=None, info=None, name='Generic Device',
 
         self.owner = owner
 
-        self.info = info
+        self.info = owner.info
+        self.shortname = owner.shortname
```

Comment:
could use a property here too

Reply:
Done.

Reply:
@FFY00: when you just implement something as suggested, best to mark the conversation as resolved as well in the interface.